content/blog/cpu-profiling-2/index.md

```diff
@@ -0,0 +1,188 @@
+---
+path: '/cpu-profiling'
+title: 'Part 2: A Simple eBPF Continuous Cross-Language (Go, Rust, C/C++) Profiler'
+date: 2021-6-1T06:00:00.000+00:00
+featured_image: hero-image.png
+categories: ['Pixie Team Blogs']
+authors: ['Omid Azizi', 'Pete Stevenson']
+emails: ['oazizi@pixielabs.ai', 'jps@pixielabs.ai']
+featured: true
+---
+
+In the last [blog post](/cpu-profiling/#part-1:-an-introduction-to-application-performance-profiling), we discussed the basics of CPU profilers for compiled languages like Go, C++ and Rust. We ended by saying we wanted a sampling-based profiler that met these two requirements:
+
+1. Does not require recompilation or redeployment: This is critical to Pixie’s auto-telemetry approach to observability. You shouldn’t have to instrument or even re-run your application to get observability.
+
+2. Has very low overheads: This is required for an always-on profiler, which was desirable for making performance profiling as low-effort as possible.
```

Comment:
Let's change `always-on` to `continuous (always-on)`, because we use the term continuous in the title.